packages/world/remappings.txt

```diff
@@ -0,0 +1,3 @@
+ds-test/=../../node_modules/ds-test/src/
+forge-std/=../../node_modules/forge-std/src/
+@latticexyz/store/=../../node_modules/@latticexyz/store/src/
```

Comment:
wdyt about `@latticexyz/store/src/=../../node_modules/@latticexyz/store/src/`?
(or just `@latticexyz/=../../node_modules/@latticexyz/`)
The extra `src/` is only 4 symbols, but this essentially gets rid of remappings.
It could even truly get rid of remappings if using `include_path=../../node_modules` in foundry.toml (except it freaks out in some edge cases with nested deps, but eventually it might work well).